src/library/dao/cuedao.cpp

```diff
@@ -44,7 +44,7 @@ CuePointer cueFromRow(const QSqlRecord& row) {
     const auto position =
             mixxx::audio::FramePos::fromEngineSamplePosMaybeInvalid(
                     row.value(row.indexOf("position")).toDouble());
-    int lengthFrames = row.value(row.indexOf("length")).toInt() / mixxx::kEngineChannelCount;
+    FrameDiff_t lengthFrames = row.value(row.indexOf("length")).toDouble() / mixxx::kEngineChannelCount;
```

Comment:
`‘FrameDiff_t’ was not declared in this scope; did you mean ‘mixxx::audio::FrameDiff_t’?`

Reply:
fixed